src/lessons/02_introduction_to_frontend/introduction-to-frontend.slides.html

```diff
@@ -0,0 +1,337 @@
+              <html>
+                <head>
+                  <title>My first website</title>
+                </head>
+                <body>
+
+                </body>
+              </html>
+            </script></code></pre>
+
+            <p style="font-size: .3em;" class="current-visible">
+              👉 the &lt;body&gt; tag contains the content and its structure of your entire web page.
+            </p>
+          </section>
+          <section data-auto-animate>
+            <pre data-id="code-animation"><code data-trim class="html"><script type="text/template">
+              <!DOCTYPE html>
+              <html>
+                <head>
+                  <title>My first website</title>
+                </head>
+                <body>
+                  <h1>Welcome!</h1>
+                </body>
+              </html>
+            </script></code></pre>
+
+            <p style="font-size: .3em;" class="current-visible">
+              👉 the &lt;h1&gt; represents the level of heading of your web page. (we'll dip dive on this)
+            </p>
+          </section>
+          <section data-auto-animate>
+            <pre data-id="code-animation"><code data-trim class="html"><script type="text/template">
+              <!DOCTYPE html>
+              <html>
+                <head>
+                  <title>My first website</title>
+                </head>
+                <body>
+                  <h1>Welcome!</h1>
+                  <p>This is my first HTML site.</p>
+                </body>
+              </html>
+            </script></code></pre>
+
+            <p style="font-size: .3em;" class="current-visible">
+              👉 the &lt;p&gt; is simply a paragraph text that can be displayed at anywhere on your page. 🙂
+            </p>
+          </section>
+        </section>
+
+        <!--SLIDE 5-->
+        <!--Inspecting elements (introduction)-->
+        <section>
+          <!--SLIDE 5.1-->
+          <section data-auto-animate data-timing="10">
+            <h2>Inspecting elements</h2>
+          </section>
+          
+          <!--SLIDE 5.2-->
+          <section data-auto-animate data-timing="10">
+            <h2>Inspecting elements</h2>
+            <p>Why?</p>
+          </section>
+          
+          <!--SLIDE 5.3-->
+          <section data-auto-animate data-timing="300">
+            <h2>Inspecting elements</h2>
+            <p>Why?</p>
+
+            <ul style="list-style: none;">
+              <li><span class="fragment" data-fragment-index="0">✅</span> Debug <i data-fragment-index="1" class="fragment r-stretch"><small>👉 stands for identifying and removing errors from code</small></i></li>
+              <li><span class="fragment" data-fragment-index="2">✅</span> Re-order elements</li>
+              <li><span class="fragment" data-fragment-index="3">✅</span> Manipulate elements</li>
+              <li><span class="fragment" data-fragment-index="4">☑️</span> Do that together</li>
```

Comment:
Just a quick tip: You can save yourself some typing: you only need `data-fragment-index` if the fragments are supposed to appear in a different order than from top to bottom.

Reply:
Oh nice. This case I also need it in a different order since I want only the "✅" to appear after each other and not the whole "**li**".

Reply:
That, too, works. The only thing that matters is the order in which they appear in html

Reply:
Oh, beautiful <3 

Will change that, tnxxxx